src/routes/contact-us/+page.svelte

```diff
@@ -0,0 +1,33 @@
+<script>
+	import fb_logo from './Facebook_Logo_Primary.png';
+	import ig_logo from './Instagram_Glyph_Gradient.png';
+</script>
+
+<h1>Contact Us</h1>
+
+<p>
+	Email Address: <a href="mailto:kendoinfo@umich.edu"> kendoinfo@umich.edu</a>
+</p>
+
+<h2>Find us on:</h2>
+
+<figure>
+	<a href="https://www.facebook.com/groups/1038072369705184">
+		<img class="social-media" src={fb_logo} alt="Facebook logo" />
+	</a>
+	<a href="https://www.instagram.com/kendoumich/">
+		<img class="social-media" src={ig_logo} alt="Instagram logo" />
+	</a>
+</figure>
+
+<style>
+	.social-media {
+		padding: 1em;
```

Comment:
Why are we using `em` all of a sudden?

Reply:
Idk. I was just trying to go as fast as possible while making things look somewhat bearable.

Reply:
Ok sounds good let's at least use consistent units though.

Reply:
Fixed in [1556dbe](https://github.com/kendoclub-at-umich/website/pull/32/commits/1556dbe5c23b420193744c94f157f4d08946c085)